test/utilitiesTest.js

```diff
@@ -34,30 +34,38 @@ describe('requestedDataIsHistoric_', () => {
   })
 
   it('returns true when period is over HISTORIC_DATA_THRESHOLD', () => {
-    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD)
-
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
     expect(san.requestedDataIsHistoric_(from)).to.be.true
   })
 })
 
 describe('assertCanAccessHistoricData_', () => {
   it('throws an error when historic data is requested and api key is not present', () => {
-    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD)
+    const from = subDays(now, san.HISTORIC_DATA_THRESHOLD + 1)
+    const firstSlug = 'ethereum'
```

Comment:
`firstSlug`, `secondSlug` ... are not good names for variables.
Define two consts at the top of the file:
```javascript
const slug = 'santiment'
const ethereumSlug = 'ethereum'
```
and use them in all tests from this file.